server/api/helpers/encryption/encrypt.js

```diff
@@ -0,0 +1,21 @@
+import bcrypt from 'bcrypt';
```

Comment:
@Ibidapo Why do we have this file back in the project ?

Reply:
The file must have been added while rebasing